src/Response/ChangeAccountKeyResponse.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright (C) 2018 Adshares sp. z o.o.
+ *
+ * This file is part of ADS PHP Client
+ *
+ * ADS PHP Client is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * ADS PHP Client is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with ADS PHP Client.  If not, see <https://www.gnu.org/licenses/>
+ */
+
+namespace Adshares\Ads\Response;
+
+/**
+ * Response for `change_account_key` transactions.
+ *
+ * @package Adshares\Ads\Response
+ */
+class ChangeAccountKeyResponse extends AbstractResponse
+{
+    /**
+     * `true`, if account key was changed, `false` otherwise
+     *
+     * @var bool
+     */
+    protected $keyChanged;
+
+    /**
+     * @param array $data
+     */
+    protected function loadData(array $data): void
+    {
+        parent::loadData($data);
+
+        $this->keyChanged = array_key_exists('result', $data) ? 'PKEY changed' === $data['result'] : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1125)